osu/osu.py

```diff
@@ -47,6 +46,7 @@ def __init__(self):
             embed.add_field(name="Playcount", value=osu[0]["playcount"])
             embed.add_field(name="Total score", value=osu[0]["total_score"])
             embed.add_field(name="Total seconds played", value=osu[0]["total_seconds_played"])
+            embed.set_thumbnail(url="https://a.ppy.sh/{}".format(osu[0]["user_id"]))
```

Comment:
This assumes that there is *always* a picture, can you confirm that's true?

Reply:
I can confirm this is true. As long as the user_id is a valid one, which it would be, it will take the picture directly from their profile.

Reply:
Alright